simulatingrisk/hawkdovevar/model.py

```diff
@@ -113,3 +114,78 @@ def adjustment_round(self) -> bool:
             and self.schedule.steps > 0
             and self.schedule.steps % self.adjust_round_n == 0
         )
+
+    def get_data_collector_options(self):
+        # in addition to common hawk/dove data points,
+        # we want to include population risk category
+        opts = super().get_data_collector_options()
+        opts["model_reporters"]["population_risk_category"] = "population_risk_category"
```

Comment:
double key assignments in dicts always scare me a bit, in case the first key returns a KeyError, but it looks like the opts dict is guaranteed to contained the model_reporters key.